tmk_core/protocol/host.c

```diff
@@ -133,6 +146,15 @@ void host_system_send(uint16_t usage) {
     if (usage == last_system_usage) return;
     last_system_usage = usage;
 
+#ifdef BLUETOOTH_ENABLE
+    send_output_t where_to_send = get_send_output();
+    if ((where_to_send == SEND_OUTPUT_BLUETOOTH) || (where_to_send == SEND_OUTPUT_BOTH)) {
+        bluetooth_driver.send_system(usage);
```

Comment:
If `send_system` is intended to be optional, this needs a NULL check.

Reply:
Yep.  Nice catch ... and also `task` in *`keyboard.c`*.  TY.